website/docs/integrations/sources/github/index.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Github
+---
+
+Allows users to authenticate using their Github credentials
+
+## Preparation
+
+The following placeholders will be used:
+
+- `authentik.company` is the FQDN of the authentik install.
+- `www.my.company` Homepage URL for your site
+
+## Github
+
+1. Create an OAuth app under Developer Settings https://github.com/settings/developers by clicking on the **Register a neww application**
+
+![Register OAuth App](githubdeveloper1.png)
+
+2. **Application Name:** <Choose a name users will recognize ie: Authentik>
```

Comment:
Since HTML is allowed in markdown, < and > are reserved characters. I think in this case the chars can just be removed, and for the other cases, where another field is referenced, you can do 
```
`<foo>`
```

Reply:
Sounds Good.  I'll update the pages now.

Reply:
Formatting has been cleaned up